app/controllers/application_controller.rb

```diff
@@ -2,10 +2,15 @@ class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
   before_action :prepare_modal_donation
+  before_action :prepare_modal_event
 
   private
 
   def prepare_modal_donation
     @modal_donation = Donation.new
   end
+
+  def prepare_modal_event
```

Comment:
Unless we should be able to create events from every page, let's move this to a more specific controller. 😀